src/style.css

```diff
@@ -1,3 +1,88 @@
+html {
+  /* stylelint-disable-next-line */
+  font-family:
+  Roboto, Arial,
+  Helvetica Neue,
+  Helvetica,
+  sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --active-link-color: #00acdc;
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  line-height: 0;
+}
+
+.figure__logo-moyo {
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:first-of-type) {
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). Here, the `:not(:first-of-type)` pseudo-class is used to add a left margin, which is horizontal, so it's fine. However, if you were to add vertical margins, stick to just one direction.